src/components/Discover/TrendingTokens.tsx

```diff
@@ -581,23 +573,23 @@ export function TrendingTokens() {
           style={{ marginHorizontal: -padding }}
         >
           <CategoryFilterButton
-            category="trending"
-            label={i18n.t(t.filters.categories.trending)}
+            category={TrendingCategory.Trending}
+            label={i18n.t(t.filters.categories.TRENDING)}
```

Comment:
I personally don't like i18n keys being uppercase and think they should follow either snake_case or camelCase. Nitpick though

Reply:
used uppercase to remain consistent with the type because initially was thinking of doing `t.filters.categories[TrendingCategory.Trending]`